plugins/outputs/kairosdb/kairosdb.go

```diff
@@ -0,0 +1,287 @@
+package kairosdb
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var (
+	allowedChars = regexp.MustCompile(`[^a-zA-Z0-9-_./\p{L}]`)
+	hypenChars   = strings.NewReplacer(
+		"@", "-",
+		"*", "-",
+		`%`, "-",
+		"#", "-",
+		"$", "-")
+	defaultHttpPath  = "/api/v1/datapoints"
+	defaultSeperator = "_"
```

Comment:
Those are constants. Alternatively you should use them directly at the one location where you specify those and get rid of the constant declaration.